ruby/files_and_serialization/project_event_manager.md

```diff
@@ -28,14 +28,10 @@ representatives for each attendee based on their zip code.
 
 ### Initial Setup
 
-Create a project directory folder named `event_manager` wherever you want to store
-your project. In the project directory, create another folder named `lib` and inside
-that folder create a plain text file named `event_manager.rb`. Using your command-line
-interface (CLI), you can enter the following commands:
+Create a new GitHub repository named `event_manager` and clone it to your `repos` directory. In the new `event_manager` directory, create another folder named `lib` and inside
```

Comment:
Whats your thoughts on having this as a numbered list of steps?

```suggestion
1. Create a new GitHub repository named `event_manager` and clone it to your `repos` directory. 
2. In the new `event_manager` directory, create another folder named `lib` and inside that folder create a plain text file named `event_manager.rb`. You can create the directory and file by using the following commands:
```

Reply:
That would work, in my opinion. While a little different than the normal, it still addresses the main concern I had, and follows most of the conventional TOP lesson patterns.